### DISCUSSION TITLE
Aitrack not detecting my laptop camera ?

### DISCUSSION THREAD

**Discussion by Mohamed-Sewilam** [AIRLegend/aitrack#266] [General]
Im trying to use both opentrack and ai track  but for some reason aitrack not  detecting my laptop camera when OT does can someone tell how to fix that?

**Comment by IcyMilk21** [comment]
same problem. I think aitrack can't detect laptop camera because laptop's camera isn't ir camera.(same problem in Windows Hello Face Login) I'm using Legion Slim 5 14APH8. What's your product?
